nixos/modules/programs/appimage.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, config, pkgs, ... }:
+
+let
+  cfg = config.programs.appimage;
+  package = pkgs.appimage-run.override {extraPkgs = pkgs: cfg.extraPackages;};
+in
+
+{
+  options = {
+    programs.appimage = {
```

Comment:
Suggestion: allow to override the package.

Basically `programs.appimage.package`.

Reply:
Added, but not sure how useful that really is.